stable/mongodb-replicaset/templates/mongodb-metrics-secret.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and (.Values.auth.enabled) (not .Values.auth.exisitingMetricsSecret) (.Values.metrics.enabled) -}}
+{{- if and (.Values.auth.enabled) (not .Values.auth.existingMetricsSecret) (.Values.metrics.enabled) -}}
```

Comment:
Can you fix the spelling in the values.yaml and add it to the README?

Reply:
Sure, on the way!